config/manager-full/auth_proxy_patch.yaml

```diff
@@ -8,6 +8,15 @@ metadata:
 spec:
   template:
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: beta.kubernetes.io/arch
```

Comment:
`key` should be `kubernetes.io/arch`. See k8s [labels-annotations-taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) docs as well https://github.com/kubernetes/kubernetes/pull/73333

Note that we already use those values for samba pods' selector: https://github.com/samba-in-kubernetes/samba-operator/blob/master/internal/planner/selector.go#L14